tooling/airneis/db-populator/category/category.go

```diff
@@ -0,0 +1,40 @@
+package category
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	httpclient "github.com/krispyTech/airneis/tooling/db-populator/httpClient"
+)
+
+type Category struct {
+	Name           string `json:"name"`
+	ThumbnailUrl   string `json:"thumbnailUrl"`
+	Slug           string `json:"slug"`
+	OrderOfDisplay *uint  `json:"orderOfDisplay"`
+}
+
+func CreateCategories(hc httpclient.HttpApi) error {
+	for i := 0; i < 100; i++ {
+		categoryID := uint(i)
```

Comment:
Is it necessary because if we create a const we will able to type it

Reply:
Yes it is, it's to have an autonomous creation.